buffer/src/main/java/io/netty/buffer/PoolArena.java

```diff
@@ -26,6 +26,7 @@
 import java.util.List;
 
 abstract class PoolArena<T> implements PoolArenaMetric {
+    static final boolean HAS_UNSAFE = PlatformDependent.hasUnsafe();
```

Comment:
Why not just call PlatformDependent.hasUnsafe() each time? I suspect that it's a static final as well, no?